### PR TITLE
Install openstackclient using base::openstackclient

### PR DESCRIPTION
**Pull request by ramereth**
This fixes an issue on util.

### CODE REVIEW

File: test/integration/packer_pipeline_node/serverspec/packer_pipeline_node_spec.rb [hunk 1]
```diff
@@ -51,10 +51,6 @@
   it { should be_executable }
 end
 
-describe command("scl enable python27 'openstack --version'") do
-  its(:exit_status) { should eq 0 }
-end
-
```
Comment (doublej472) [inline test/integration/packer_pipeline_node/serverspec/packer_pipeline_node_spec.rb]: This test can be adapted for the new setup, like `'/usr/local/bin/openstack --version'`

Comment (ramereth) [reply test/integration/packer_pipeline_node/serverspec/packer_pipeline_node_spec.rb]: That's already covered as an integration test in base::openstackclient

Comment (doublej472) [reply test/integration/packer_pipeline_node/serverspec/packer_pipeline_node_spec.rb]: What if someone accidentally removes it in this cookbook? or breaks it in some way?

Comment (ramereth) [reply test/integration/packer_pipeline_node/serverspec/packer_pipeline_node_spec.rb]: I suppose that's a good point, I'll adapt this test.

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.9.12.  Have a nice day!

**Pull request merged**